packages/extension-ui/src/messaging.ts

```diff
@@ -125,16 +125,17 @@ export async function createSeed (length?: SeedLengths, type?: KeypairType): Pro
   return sendMessage('pri(seed.create)', { length, type });
 }
 
+// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
This should not be needed? Await on line 141?

Reply:
absolutely, that's a stray line from tests I did. Removing it.